### PR TITLE
Kontener z modelem i API (6)

### PR DESCRIPTION
**Pull request by DawidPludowski**

### CODE REVIEW

File: containerized_ml_models/docker_api_6/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM continuumio/anaconda3
+
+COPY ./ / 
+
+RUN conda env create --name ml-env --file requirements.txt
+
+ENTRYPOINT [ "conda", "run", "-n", "ml-env", \
+    "--no-capture-output", "python", "api_predict.py" ]
```
Comment (matined) [inline containerized_ml_models/docker_api_6/Dockerfile:8]: It's a good practice to put a newline at the end of a flat file.

**Review by matined — APPROVED** [review]
Good job, I really like this solution. The only thing I would think about is whether or not should we somehow hide the model from a global scope, so that it cannot be modified.

**Comment by matined** [comment]
> @matined, thanks for review! I just added singleton implementation to model, could you check it?

Great, I've never used the `singleton_decorator` module, but it seems correct. If you've got nothing more to add please change the dictionary name from 'docker_api_6' to 'model_api_6' and we'll merge it :))

**Comment by DawidPludowski** [comment]
@matined done

**Pull request merged**